internal/output/combined.go

```diff
@@ -282,7 +287,7 @@ func Combine(inputs []ReportInput) (Root, error) {
 	combined.PastTotalMonthlyCost = pastTotalMonthlyCost
 	combined.DiffTotalHourlyCost = diffTotalHourlyCost
 	combined.DiffTotalMonthlyCost = diffTotalMonthlyCost
-	combined.TimeGenerated = time.Now().UTC()
+	combined.TimeGenerated = lastestGeneratedAt
```

Comment:
I'm not 100% sure this won't cause issues, but don't see the harm in using the lastest generated at instead of the current time for the combined run data.

Reply:
I don't see a problem as well, famous last words.